data/relup_bases.eterm

```diff
@@ -0,0 +1,32 @@
+%% -*- mode: erlang; -*-
+
+#{<<"4.4.0">> => #{from_versions => [],otp => <<"24.1.5-3">>},
+  <<"4.4.1">> => #{from_versions => [<<"4.4.0">>],otp => <<"24.1.5-3">>},
+  <<"4.4.2">> =>
+      #{from_versions => [<<"4.4.0">>,<<"4.4.1">>],otp => <<"24.1.5-3">>},
+  <<"4.4.3">> =>
+      #{from_versions => [<<"4.4.0">>,<<"4.4.1">>,<<"4.4.2">>],
+        otp => <<"24.1.5-3">>},
+  <<"4.4.4">> =>
+      #{from_versions => [<<"4.4.0">>,<<"4.4.1">>,<<"4.4.2">>,<<"4.4.3">>],
+        otp => <<"24.1.5-3">>},
+  <<"4.4.5">> =>
+      #{from_versions =>
+            [<<"4.4.0">>,<<"4.4.1">>,<<"4.4.2">>,<<"4.4.3">>,<<"4.4.4">>],
+        otp => <<"24.1.5-3">>},
+  <<"4.4.6">> =>
+      #{from_versions =>
+            [<<"4.4.0">>,<<"4.4.1">>,<<"4.4.2">>,<<"4.4.3">>,<<"4.4.4">>,
+             <<"4.4.5">>],
+        otp => <<"24.1.5-3">>},
+  <<"4.4.7">> =>
+      #{from_versions =>
+            [<<"4.4.0">>,<<"4.4.1">>,<<"4.4.2">>,<<"4.4.3">>,<<"4.4.4">>,
+             <<"4.4.5">>,<<"4.4.6">>],
+        otp => <<"24.1.5-3">>},
+  <<"4.4.8">> =>
+      #{from_versions =>
+            [<<"4.4.0">>,<<"4.4.1">>,<<"4.4.2">>,<<"4.4.3">>,<<"4.4.4">>,
+             <<"4.4.5">>,<<"4.4.6">>,<<"4.4.7">>],
+        otp => <<"24.1.5-3">>},
+  <<"4.5.0">> => #{from_versions => [<<"4.4.8">>],otp => <<"24.3.4.2-1">>}}.
```

Comment:
nl
```suggestion
  <<"4.5.0">> => #{from_versions => [<<"4.4.8">>],otp => <<"24.3.4.2-1">>}}.
```

Reply:
:heavy_check_mark: 

Added a new line to the `insert-new-vsn` command.